net/acme-common/files/acme.uci-defaults

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
+# Create a symlink to webroot
+if [ -e /www/ ] && [ ! -L /www/.well-known/acme-challenge ]; then
```

Comment:
This should probably be stricter on the type of /www and looser on the type of the acme-challenge directory. I.e., we don't want to do this if /www is really a symlink to somewhere else (since that's an indication that this is not a standard setup, so we should cancel it), and we also don't want to run this if /www/.well-known/acme-challenge already exists, but is not a link. So something like:

`if [ -d /www/ ] && [ ! -e /www/.well-known/acme-challenge ]; then`